### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by Aadi-mone** [ElCholoGamer/userlogin#23]
Can this plugin detect premium and cracked accounts?

and can avoid /login for premium players and allow /login for cracked players?

Any help really appreciated.

**Comment by ElCholoGamer** [comment]
Nope. All premium verification is done by FastLogin, and it currently doesn't provide a way to block or allow commands.

**Comment by ElCholoGamer** [comment]
Will reopen this issue if a future version of FastLogin allows this, but just gonna close the issue for now.
